ledger/participant-integration-api/src/main/scala/platform/store/DbType.scala

```diff
@@ -40,4 +40,17 @@ private[platform] object DbType {
     case _ =>
       sys.error(s"JDBC URL doesn't match any supported databases (h2, pg)")
   }
+
+  sealed trait AsyncCommitMode {
+    def setting: String
+  }
+  object SynchronousCommit extends AsyncCommitMode {
+    override val setting: String = "ON"
+  }
+  object AsynchronousCommit extends AsyncCommitMode {
+    override val setting: String = "OFF"
+  }
+  object LocalSynchronousCommit extends AsyncCommitMode {
```

Comment:
I am not sure what `LOCAL` brings to the plate here. As I understand it behaves the same as `ON` (sync commit), unless there are standby PG replicas. And in that case, do we want `LOCAL` or would we be better off with just `ON`?

Reply:
Nevermind, I just noticed the linked PR. Thanks!

Reply:
I added the following link to the description above: https://www.percona.com/blog/2020/08/21/postgresql-synchronous_commit-options-and-synchronous-standby-replication/